website/blog/2023-11-01-record-level-index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Record Level Index: Hudi's blazing fast indexing for large-scale datasets"
+excerpt: "Announcing the Record Level Index in Apache Hudi"
+author: Shiyan Xu and Sivabalan Narayanan
+category: blog
+image: /assets/images/blog/debezium.png
```

Comment:
1 nit. 
we might need to choose one of the images as the thumbnail image. 
I copied over from previous blog and missed to edit it.